### PR TITLE
add sword animation labels

### PR DESCRIPTION
**Pull request by zelosos**

### CODE REVIEW

File: src/code/bank0.asm [hunk 1]
```diff
@@ -1992,22 +1992,23 @@ UseHookshot::
 ; Inputs:
 ;   a    inventory item
 label_1321::
```
Comment (kemenaran) [inline src/code/bank0.asm:1994]: Nitpick: do you think we can rename this label `UseSword` then?

File: src/constants/gameplay.asm [hunk 2]
```diff
@@ -157,6 +157,14 @@ SWORD_DIRECTION_LEFT_TOP     equ $05
 SWORD_DIRECTION_TOP          equ $06
 SWORD_DIRECTION_RIGHT_TOP    equ $07
 
+; Values for wSwordAnimationState
+SWORD_ANIMATION_STATE_NONE         equ $00
+SWORD_ANIMATION_STATE_DRAW         equ $01
+SWORD_ANIMATION_STATE_SWING_START  equ $02
+SWORD_ANIMATION_STATE_SWING_MIDDLE equ $03
+SWORD_ANIMATION_STATE_SWING_END    equ $04
+SWORD_ANIMATION_STATE_HOLDING      equ $05
```
Comment (kemenaran) [inline src/constants/gameplay.asm:166]: 👍

File: src/code/bank0.asm (hunk 1 above)
Comment (zelosos) [reply src/code/bank0.asm:1994]: I think the same, but this label already exists. So I investigation is needed, what it does beside the other function.
This function is used inside UseSword.
So I wanted to label stuff around it to get a better view.

Comment (kemenaran) [reply src/code/bank0.asm:1994]: Seems suitable to me 👍

**Pull request merged**